src/pages/regio/index.js

```diff
@@ -42,6 +42,31 @@ function Regio() {
       : null;
   }, [router]);
 
+  const contentRef = React.useRef();
+  const selectRegioWrapperRef = React.useRef();
+
+  /**
+   * Focuses region select element. Triggered by a screen reader
+   * only button at the end of the content.
+   */
+  const focusRegioSelect = () => {
+    if (!selectRegioWrapperRef.current) return;
+    try {
+      selectRegioWrapperRef.current.querySelector('input').focus();
+    } catch {}
```

Comment:
How about something like this?
Not sure if the `.length` is a correct check in this instance, but as an example...

```suggestion
const focusPoint = selectRegioWrapperRef.current.querySelector('input');
if (focusPoint.length > 0) {
  focusPoint.focus()
}
```

Reply:
Oh yes, similar to what I wrote in the previous comment! I think this is less confusing way of doing it. You want me to change it?

Reply:
Yes please, I'd rather have more strict checks before calling a function then relying on a silent error.